editor/export/editor_export_platform.cpp

```diff
@@ -1407,6 +1407,12 @@ Error EditorExportPlatform::export_project_files(const Ref<EditorExportPreset> &
 	Vector<String> forced_export = get_forced_export_files();
 	for (int i = 0; i < forced_export.size(); i++) {
 		Vector<uint8_t> array = FileAccess::get_file_as_bytes(forced_export[i]);
+		if (GDExtension::get_extension_list_config_file() == forced_export[i]) {
+			array = _filter_extension_list_config_file(forced_export[i], paths);
+			if (array.size() == 0) {
+				continue;
+			}
+		}
```

Comment:
This ends up reading the `extension_list.cfg` twice. Maybe this could be refactored like:

```suggestion
		Vector<uint8_t> array;
		if (GDExtension::get_extension_list_config_file() == forced_export[i]) {
			array = _filter_extension_list_config_file(forced_export[i], paths);
			if (array.size() == 0) {
				continue;
			}
		} else {
			array = FileAccess::get_file_as_bytes(forced_export[i]);
		}
```

Reply:
Yes, that seems like it would be more efficient in the case where the extension list config file is being updated.  

I'm not sure there is a performance difference between constructing the array from an other array at the original line 1409, and doing an empty initialization and assignment at the new 1409 and 1416.  Almost all files being copied are NOT the extension list config file, so line 1409 and then 1416 in your revised version (doing an assignment into the array after previous initialization) is the code executed for almost every file.

I think when I wrote it originally I was going for the least impact to existing code, and only having extra code being executed for the one extension list config file, and hoping that would make the merge easier.

If you or other reviewers think it is worth updating and redoing the pull request, I'll work on it later this week.

Reply:
Took a while for me to get back to it, but I implemented the requested change and fixed a merge problem.  Hopefully this will allow it to make it into 4.4